src/services/midgard/types.ts

```diff
@@ -23,15 +23,10 @@ export type PoolsState = {
   assetDetails: AssetDetails
   poolAssets: PoolAssets
   poolDetails: PoolDetails
+  pricePools: PricePools
+  selectedPricePool: PricePool
```

Comment:
Extract `selectedPricePool`  from `PoolsState`. No need to have it wrapped in `RemoteData` as we do it with `PoolsState` by using `PoolsStateRD`. That will be another PR.

Reply:
https://github.com/thorchain/asgardex-electron/issues/185